examples/official-storybook/stories/addon-docs/container-override.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { DocsContainer } from '@storybook/addon-docs/blocks';
+
+export default {
+  title: 'Addons/Docs/container-override',
+  parameters: {
+    docs: {
+      // eslint-disable-next-line react/prop-types
+      container: ({ children, context }) => (
+        <DocsContainer context={context}>
+          <div style={{ border: '5px solid red' }}>{children}</div>
+        </DocsContainer>
```

Comment:
Does this story make use of `AddContext` at all?

Reply:
No, but thanks to your other fix I was able to convert the file to MDX and now it does! 🙏